app/models/recipe.rb

```diff
@@ -0,0 +1,3 @@
+class Recipe < ApplicationRecord
+    validates :name, presence:true
```

Comment:
Should put a space after the colon
```suggestion
    validates :name, presence: true
```